.github/ISSUE_TEMPLATE/feature-enhancement-request.md

```diff
@@ -0,0 +1,16 @@
+---
+name: Feature enhancement request
+about: Suggest an idea for ytt
+labels: carvel-triage
```

Comment:
We should also add `enhancement` when it is opened.

Reply:
done